src/wolfio.c

```diff
@@ -59,6 +66,11 @@ Possible IO enable options:
    but they'll still need SetCallback xxx() at end of file
 */
 
+#if defined(NO_ASN_TIME) && defined(DTLS_RECEIVEFROM_TIMEOUT_ON_MULTIPLE_INVALID_PEER) \
+  && !defined(DTLS_RECEIVEFROM_MAX_INVALID_PEER)
+#define DTLS_RECEIVEFROM_MAX_INVALID_PEER 10
+#endif
```

Comment:
I think this should be enabled by default with `#ifndef NO_ASN_TIME`. The `DTLS_RECEIVEFROM_TIMEOUT_ON_MULTIPLE_INVALID_PEER` macro should be changed to something that the user can define to *disable* the return. We don't want to provide security by default.